sdk/storage/Azure.Storage.Files.Shares/tests/ServiceClientTests.cs

```diff
@@ -105,6 +105,7 @@ public async Task GetPropertiesAsync_Error()
 
         [Test]
         [NonParallelizable]
+        [Ignore("Need to re-record")]
```

Comment:
Need to re-record before merging.

Reply:
Can't re-record because it appears SMB multi-channel isn't fully rolled out.  Marking as ignore for now.